On Mac, the ./build.sh doesn't finish and ends with 1 error generated.

Could you please change the installation instructions for Mac because of this error?

```
#include <gtest/gtest.h>
         ^~~~~~~~~~~~~~~
1 error generated.
[106/134] Compiling C++ ob.../src_chess_board_test.cc.
FAILED: chessboard_test.p/src_chess_board_test.cc.o 
c++ -Ichessboard_test.p -I. -I../.. -I../../src -flto -Xclang -fcolor-diagnostics -DNDEBUG -pipe -Wall -Winvalid-pch -Wnon-virtual-dtor -Wextra -Wpedantic -std=c++17 -O3 -Wthread-safety -march=native '-DBUILD_IDENTIFIER="git.unknown"' -DUSE_BLAS -DNO_PEXT -isystem../../third_party -MD -MQ chessboard_test.p/src_chess_board_test.cc.o -MF chessboard_test.p/src_chess_board_test.cc.o.d -o chessboard_test.p/src_chess_board_test.cc.o -c ../../src/chess/board_test.cc
../../src/chess/board_test.cc:21:10: fatal error: 'gtest/gtest.h' file not found
#include <gtest/gtest.h>
         ^~~~~~~~~~~~~~~
1 error generated.
[121/134] Linking target lc0
```

Thanks in advance!

--- a/README.md
+++ b/README.md
@@ -166,12 +166,26 @@ Or.
  * Install Xcode command-line tools: ``xcode-select --install``
  * Install "XCode Developer Tools" through the app store. (First one on the list of Apps if searched.)
  * Associate the SDK headers in XCode with a command: export CPATH=\`xcrun --show-sdk-path\`/usr/include
+ 
+6. Optionally install googletest (aka gtest)
+
+```
+cd ~
+git clone https://github.com/google/googletest.git
+cd googletest
+mkdir install
+cd install
+cmake ..
+make
+make install
+```
 
 Now download the lc0 source, if you haven't already done so, following the instructions earlier in the page.
 
-6. Go to the lc0 directory.
-7. Run `./build.sh` (needs step 5)
-8. The resulting binary will be in build/release
+7. Go to the lc0 directory.
+8. Run `./build.sh` (needs step 5)
+ * If you did not install gtest in step 6 then run `./build.sh -Dgtest=false` instead.
+9. The resulting binary will be in build/release
 
 ### Raspberry Pi
 

--- a/README.md
+++ b/README.md
@@ -167,25 +167,10 @@ Or.
  * Install "XCode Developer Tools" through the app store. (First one on the list of Apps if searched.)
  * Associate the SDK headers in XCode with a command: export CPATH=\`xcrun --show-sdk-path\`/usr/include
  
-6. Optionally install googletest (aka gtest)
-
-```
-cd ~
-git clone https://github.com/google/googletest.git
-cd googletest
-mkdir install
-cd install
-cmake ..
-make
-make install
-```
-
 Now download the lc0 source, if you haven't already done so, following the instructions earlier in the page.
 
-7. Go to the lc0 directory.
-8. Run `./build.sh` (needs step 5)
- * If you did not install gtest in step 6 then run `./build.sh -Dgtest=false` instead.
-9. The resulting binary will be in build/release
+6. Go to the lc0 directory.
+7. Run `./build.sh -Dgtest=false` (needs step 5)
 
 ### Raspberry Pi
 

